Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -1617,7 +1617,7 @@ void WebProcessPool::startMemorySampler(const double interval)
         // For WebProcess
         SandboxExtension::Handle sampleLogSandboxHandle;
         WallTime now = WallTime::now();
-        auto sampleLogFilePath = makeString("WebProcess", static_cast<unsigned long long>(now.secondsSinceEpoch().seconds()));
+        auto sampleLogFilePath = makeString("WebProcess"_s, static_cast<unsigned long long>(now.secondsSinceEpoch().seconds()));
```

Comment:
I wonder if we could write this as `now.secondsSinceEpoch().secondsAs<uint64_t>()` instead?